state/factory/workingset.go

```diff
@@ -715,6 +725,16 @@ func (ws *workingSet) ValidateBlock(ctx context.Context, blk *block.Block) error
 		}
 	}
 	if fCtx.EnableBlobTransaction {
+		blobTxCnt := uint64(0)
+		blobTxLimit := genesis.MustExtractGenesisContext(ctx).BlobActionLimitPerBlock
+		for _, selp := range blk.Actions {
+			if len(selp.BlobHashes()) > 0 {
+				blobTxCnt++
+				if blobTxCnt > blobTxLimit {
+					return errors.New("too many blob transactions in a block")
+				}
+			}
+		}
```

Comment:
also block the 3 types of tx before hard-fork?

Reply:
already checked in generic_validator